tests-legacy/PrestaShopBundle/Routing/Converter/RouterProviderTest.php

```diff
@@ -169,6 +169,22 @@ public function testGetLegacyRouteByAction()
                     'AdminCategories:create',
                 ],
             ],
+            [
```

Comment:
Good thing that you added a test, I think you should add one for `CacheProvider` as well since it extends the same abstract class